playground/keycloak/Keycloak.AppHost/Program.cs

```diff
@@ -0,0 +1,14 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var keycloak = builder.AddKeycloak("keycloak")
+                      .WithDataVolume();
+
+var apiService = builder.AddProject<Projects.Keycloak_ApiService>("apiservice")
+                        .WithReference(keycloak);
+
+builder.AddProject<Projects.Keycloak_Web>("webfrontend")
+    .WithExternalHttpEndpoints()
+    .WithReference(keycloak)
+    .WithReference(apiService);
+
+builder.Build().Run();
```

Comment:
I pulled down the branch and tried to run the playground sample. Does it work locally for you (anyone?).

Reply:
![image](https://github.com/dotnet/aspire/assets/513398/d68bbd98-9719-4c4d-a7f0-491137564559)

Reply:
> ![image](https://private-user-images.githubusercontent.com/513398/342343215-d68bbd98-9719-4c4d-a7f0-491137564559.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyNDk5OTEsIm5iZiI6MTcxOTI0OTY5MSwicGF0aCI6Ii81MTMzOTgvMzQyMzQzMjE1LWQ2OGJiZDk4LTk3MTktNGM0ZC1hN2YwLTQ5MTEzNzU2NDU1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyNFQxNzIxMzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YzM0NTBmNjg0ZTE4YTA4ZDRiZGFjYzgzOTBmYzQ1NTU1MWVhOTgwMWY5M2YxZmE1Mjc5NTFmMTQ0MzZmZDhiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.tZRU58Wz0FowAE0CKnTkhf2P9EK8XL0TyEA6-p2d2i0)

The realm configuration is not being imported, and, thus, the realm doesn't exist.

Reply:
I can't see that screenshot. is that file sensitive? We like to make sure things in the playground are immediately runnable.

Reply:
If the file is checked in anyway we should update the playground app to auto-import it on launch.

Reply:
Added an API to import the realm on launch.